magento2/usr/local/share/magento2/magento_functions.sh

```diff
@@ -376,6 +394,10 @@ function do_magento_download_magerun2() {
 
 function do_magento2_templating() {
   mkdir -p /app/app/etc/
+  chown -R "${CODE_OWNER}:${CODE_GROUP}" /app/app/
+  if [ "${APP_USER_LOCAL}" == "true" ]; then
+    chown "${CODE_OWNER}:${CODE_GROUP}" /app/app/etc/env.php
```

Comment:
I'm confused why this is needed when the chown above is recursive and a parent directory